src/cascadia/QueryExtension/ExtensionPalette.cpp

```diff
@@ -336,12 +336,19 @@ namespace winrt::Microsoft::Terminal::Query::Extension::implementation
             return false;
         }
         const auto contentFilters = contentFiltersObject.GetNamedObject(L"content_filter_results");
+        if (!contentFilters.HasKey(L"jailbreak"))
+        {
+            return false;
+        }
         for (const auto filterPair : contentFilters)
         {
             const auto filterLevel = filterPair.Value().GetObjectW();
-            if (filterLevel.GetNamedString(L"severity") != acceptedSeverityLevel)
+            if (filterLevel.HasKey(L"severity"))
             {
-                return false;
+                if (filterLevel.GetNamedString(L"severity") != acceptedSeverityLevel)
```

Comment:
This (both if conditions) can be expressed more easily with the 2-param version of `GetNamedString`:
```cpp
if (filterLevel.GetNamedString(L"severity", {}) != acceptedSeverityLevel)
```

It may be that the compiler can't infer the type for the 2nd param so you may have to write `winrt::hstring{}` or something. See here: https://learn.microsoft.com/en-us/uwp/api/windows.data.json.jsonobject.getnamedstring?view=winrt-22621#windows-data-json-jsonobject-getnamedstring(system-string-system-string)